sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3658,89 +3658,101 @@
           "persistent": true,
-          "hidden": false,
-          "clearOnHide": true,
-          "refreshOn": "",
-          "redrawOn": "",
-          "modalEdit": false,
-          "dataGridLabel": false,
-          "labelPosition": "top",
-          "description": "",
-          "errorLabel": "",
-          "tooltip": "",
-          "tabindex": "",
-          "autofocus": false,
+          "inputFormat": "html",
+          "protected": false,
           "dbIndex": false,
+          "case": "",
+          "truncateMultipleSpaces": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
           "customDefaultValue": "",
           "calculateValue": "",
           "calculateServer": false,
-          "widget": {
-            "type": "input"
-          },
+          "allowCalculateOverride": false,
           "validateOn": "change",
           "validate": {
             "required": false,
+            "minLength": "",
+            "maxLength": "",
+            "minWords": "",
+            "maxWords": "",
+            "pattern": "",
+            "customMessage": "",
             "custom": "",
             "customPrivate": false,
+            "json": "",
             "strictDateValidation": false,
             "multiple": false,
-            "unique": false,
-            "minLength": "",
-            "maxLength": "",
-            "pattern": "",
-            "minWords": "",
-            "maxWords": ""
+            "unique": false
           },
+          "unique": false,
+          "errorLabel": "",
+          "errors": "",
+          "key": "textArea",
+          "tags": [],
+          "properties": {},
           "conditional": {
             "show": null,
             "when": null,
-            "eq": ""
+            "eq": "",
+            "json": ""
+          },
+          "customConditional": "",
+          "logic": [],
+          "attributes": {
+            "style": "max-height: 200px; overflow: auto"
```

Comment:
if it was add in this PR. pls, move to `.scss`

Reply:
Not part of this PR